assets/blocks/email-editor.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+
+addFilter(
+	'blocks.registerBlockType',
+	'sensei-lms/email-blocks',
+	removeIrrelevantSettings
+);
+
+/**
+ * Update the blocks to remove extra settings for emails.
+ *
+ * @param {Object} settings Block settings.
+ */
+function removeIrrelevantSettings( settings ) {
+	if (
+		settings &&
+		settings.supports &&
+		settings.supports.typography &&
+		settings.supports.typography.__experimentalFontFamily
```

Comment:
I think you can use the loadash `has(setting, 'supports.typograph.__experimentalFontFamily')` 
&&
`has(settings, 'supports.typograph.fontFamily')`  to support the nonexperimental version.

Reply:
Good suggestion, thanks. Updated here https://github.com/Automattic/sensei/pull/6537/commits/38daaf7fbc3523472cf99e927ce3e3555322fd29